voluncheer/profiles/test_views.py

```diff
@@ -0,0 +1,67 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from profiles.models import Organization
+from profiles.models import User
+from profiles.models import UserType
+from profiles.models import Volunteer
+
+
+class SignupViewTest(TestCase):
```

Comment:
Would you like to move the url from voluncheer to profiles as well, since these tests are built in voluncheer?

Reply:
We can change in a follow-up pull request, will add an issue.

Reply:
Sounds good.